src/admin/secureHeader.mw.ts

```diff
@@ -13,11 +13,14 @@ export interface SecureHeaderMiddlewareCfg extends RequireAdminCfg {
  * Throws Error401Admin otherwise.
  */
 export function createSecureHeaderMiddleware(cfg: SecureHeaderMiddlewareCfg): AdminMiddleware {
-  return () => requireSecureHeaderOrAdmin(cfg)
+  return reqPermissions => requireSecureHeaderOrAdmin(cfg, reqPermissions)
```

Comment:
I'm not 100% sure it's needed, but I'm worrying that `reqPermissions` input parameter will be typed as `any` for the consumer, so, suggesting this:

```suggestion
  return (reqPermissions?: string[]) => requireSecureHeaderOrAdmin(cfg, reqPermissions)
```